plugins/permissions/src/editMember.ts

```diff
@@ -55,6 +55,19 @@ export default function editMember(bot: BotWithCache) {
       }
     }
 
+    if (options.communicationDisabledUntil) {
+      const guild = bot.guilds.get(guildId);
+      const member = bot.members.get(BigInt(`${memberId}${guildId}`));
+      if (memberId === guild?.ownerId) throw new Error("The owner of a guild cannot be timed out.");
+
+      // TODO error message
+      if (!member?.permissions) throw new Error("member?.permissions is undefined");
```

Comment:
This will always be undefined since permissions is only available for interaction members

Reply:
@itohatweb then how tf do I get the member permissions???

Reply:
Calculate using the member in cache